contracts/src/xYield/xYieldVault.sol

```diff
@@ -307,6 +307,9 @@ contract xYieldVault is ERC4626, Ownable2Step, ReentrancyGuard, Pausable {
         if (_targetChain != orderData.destinationDomain) revert InvalidOrderData();
         if (_amount != orderData.amountIn) revert InvalidOrderData();
 
+        isActiveChain = false;
+        virtualTotalAssets = 0;
```

Comment:
this needs to be `virtualTotalAssets -= _amount`

Reply:
Done

Reply:
That's actually not the correct change. I edited the message a minute after I sent it.

Please do allow me to resolve comments that I open as a reviewer, as per our defined process.